Frameworks/UIKit/NSString+UIKitAdditions.mm

```diff
@@ -267,9 +267,10 @@ - (CGSize)_sizeWithAttributes:(NSDictionary<NSString*, id>*)attributes constrain
 }
 
 // Private helper that converts a UILineBreakMode -> NSParagraphStyle
-static inline NSParagraphStyle* _paragraphStyleWithLineBreakMode(UILineBreakMode lineBreakMode) {
-    NSMutableParagraphStyle* ret = [NSMutableParagraphStyle new];
-    ret.lineBreakMode = lineBreakMode;
+static inline StrongId<NSParagraphStyle> _paragraphStyleWithLineBreakMode(UILineBreakMode lineBreakMode) {
```

Comment:
It would be clearer to just autorelease this rather than returning a StrongId.  It seems I thought [new] already did autorelease, my bad.